contributors/devel/sig-architecture/api_changes.md

```diff
@@ -1182,10 +1182,13 @@ provides some details
     }
     ```
 
+    Because validation is always on, the field can be changed from a previously invalid value to a valid value, or cleared, even if the feature gate is disabled, but the field cannot be updated to be invalid, and an invalid value will prevent other changes to the object.
```

Comment:
`even if the feature gate is disabled,` may be better in `(` and `)`?

question on : "will prevent" ... apiserver will discard any changes? (or return error?)

Reply:
Agree on the parenthetical.

Why the cutesy indent?